conmon-rs/common/proto/conmon.capnp

```diff
@@ -4,10 +4,12 @@ interface Conmon {
     ###############################################
     # Version
     struct VersionRequest {
+        metadata @1 :Data;
```

Comment:
nit... I think we should preserve the ordering within the file on all the added metadata, so it's easier to add new fields later.

Reply:
Fixed as suggested